modules/Microsoft.ServiceFabric/clusters/readme.md

```diff
@@ -248,11 +248,11 @@ The following module usage examples are retrieved from the content of the files
 
 ```bicep
 module clusters './Microsoft.ServiceFabric/clusters/deploy.bicep' = {
```

Comment:
Must be updated if the `deploy.test.bicep` tests are changed. I'd recommend to re-run the ReadMe script once in the end